Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2318,7 +2365,10 @@ - (void)_completeTextManipulation:(_WKTextManipulationItem *)item completion:(vo
             }
         })());
         auto item = items[coreFailure.index];
-        ASSERT(coreTextManipulationItemIdentifierFromString(item.identifier) == coreFailure.identifier);
+#if ASSERT_ENABLED
+        auto identifiers = coreTextManipulationItemIdentifierFromString(item.identifier);
+        ASSERT(identifiers && identifiers->frameID == coreFailure.frameID && identifiers->itemID == coreFailure.identifier);
```

Comment:
I think we probably want three asserts, otherwise it seems hard to tell which condition fails

Reply:
Sure, fixed.